docs/_substitutions.rst

```diff
@@ -0,0 +1,40 @@
+.. |bibliography| replace:: :ref:`bibliography`\
+.. |glossary| replace:: :ref:`glossary`\
+.. |inf| replace:: `~numpy.inf`
+.. |minpython| replace:: 3.8
+.. |nan| replace:: `~numpy.nan`
+.. |ndarray| replace:: :class:`~numpy.ndarray`
+.. |Quantity| replace:: :class:`~astropy.units.Quantity`
+.. |TimeDelta| replace:: :class:`~astropy.time.TimeDelta`
+.. |Time| replace:: :class:`~astropy.time.Time`
+
+.. A workaround for nested inline literals so that the filename will get
+   formatted like a file but will be a link. In the text, these get used
+   with the syntax for a substitution followed by an underscore to
+   indicate that it's for a link: |docs/_static|_
+
+.. For these workarounds, if the replacement is something in single back
+   ticks (e.g., `xarray`), then it should also be added to
+   nitpick_ignore_regex in docs/conf.py so that it doesn't get counted
+   as an error in a nitpicky doc build (e.g., tox -e doc_build_nitpicky).
+
+.. _`docs/_static`: https://github.com/PlasmaPy/PlasmaPy/tree/main/docs/_static
+.. |docs/_static| replace:: :file:`docs/_static`
```

Comment:
These substitutions & links are a workaround for allowing us to format something like a file, but also to get a link to it. There isn't really a natural way to do this in reStructuredText, unfortunately, so this is kind of a mess.